README.md

```diff
@@ -29,6 +30,109 @@ and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
 If you want to jump in early, you can try building latest code yourself.
 
+## SuperBench capabilities, workflow and benchmarking metrics
+
+The following graphic shows the capabilities provide by SuperBench core framework and its extension.
+
+<img src="imgs/superbench_structure.png">
```

Comment:
does simulation fit?

Reply:
We can show it in the picture as our related work or extension of SuperBench.